docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+that can only be completed using its private key. The bot returns the solved
+challenge, attesting to its own identity, and is conditionally allowed to join
+the cluster.
+
+As self attestation is inherently less secure than the external verification
+that would be provided by a cloud provider like AWS or a dedicated TPM, bound
+keypair joining enforces a number of additional checks to prevent abuse,
+including:
+- Join state verification to ensure the keypair cannot be usefully shared or
+  duplicated
+- Certificate generation counter checks to ensure regular bot certificates
+  cannot be usefully shared or duplicated
+- Configurable limits on how often - if at all - bots may be allowed to recover
+  using this keypair.
+
+An important benefit to Bound Keypair joining is that all joining restrictions
+can be reconfigured at any time, and bots that expire or go offline can be
+recovered by making a server-side exemption without any client-side
+intervention.
+
+Refer to the [admin guide][guide] for further details on how this join method
+works.
+
+## Prerequisites
+
+{/* note: consider edition-prereqs-tabs.mdx include for v19; it is misleading due to the minor launch release */}
+
+- A running Teleport cluster version 18.1.0 or above.
+- The `tsh` and `tctl` clients
+- (!docs/pages/includes/tctl.mdx!)
+- This guide assumes the bot node has mutable persistent storage for internal
+  bot data. While it is possible to use Bound Keypair joining can on immutable
+  hosts (like CI runs), doing so will reduce security guarantees; see the
+  [admin guide][guide] for further information.
+
+## Step 1/5. Install `tbot`
+
+**This step is completed on the bot node.**
+
+First, `tbot` needs to be installed on the node that you wish to use Machine ID
+on.
+
+Download and install the appropriate Teleport package for your platform:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 2/5. Create a Bot
+
+**This step is completed on your local machine.**
+
+(!docs/pages/includes/machine-id/create-a-bot.mdx!)
+
+## Step 3/5. Create a join token
+
+**This step is completed on your local machine.**
+
+In this guide, we'll demonstrate joining a bot using a registration secret: this
+is a one-time use secret the bot can provide to Teleport to authenticate its
+first join. Once authenticated, the bot automatically generates a keypair and
+registers its public key with Teleport for use in all future join attempts.
+
+Create `token-example.yaml`:
+
+```yaml
+kind: token
+version: v2
+metadata:
+  # This name will be used in tbot's `onboarding.token` field.
+  name: example
+spec:
+  roles: [Bot]
+  # bot_name should match the name of the bot created earlier in this guide.
+  bot_name: example
+  join_method: bound_keypair
+  bound_keypair:
+    recovery:
+      mode: standard
+      limit: 1
+```
+
+Replace:
+- `example` in `spec.bot_name` with the name of the bot you created in the
+  second step
```

Comment:
I would just write this as a plain english sentence. A list containing one item looks a little awkward to me.

```suggestion
Replace `example` in `spec.bot_name` with the name of the bot you created in the second step.
```